resources/src/idea/JavaActions.xml

```diff
@@ -127,7 +124,7 @@
       <add-to-group group-id="XDebugger.Frames.Tree.Popup"/>
       <add-to-group group-id="XDebugger.Actions"/>
     </action>
-    <action id="DumpThreads" class="com.intellij.debugger.actions.ThreadDumpAction" text="Get thread dump" icon="AllIcons.Actions.Dump">
+    <action id="DumpThreads" class="com.intellij.debugger.actions.ThreadDumpAction" icon="AllIcons.Actions.Dump">
```

Comment:
Hi there,

Will you be using anything else from this PR or can I close it?

Regards,

Sérgio Ribeiro
Porto - Portugal